.github/actions/maven-build-and-tag/action.yml

```diff
@@ -222,14 +225,47 @@ runs:
         echo "testcontainers.reuse.enable=true" > ~/.testcontainers.properties
         echo "TESTCONTAINERS_RYUK_DISABLED=true" >> $GITHUB_ENV
 
+    - name: Set skip tests env variable
+      id: set-skip-tests-env-variable
+      if: github.event_name == 'pull_request'
+      env:
+        SKIP_TESTS: ${{ contains(github.event.pull_request.labels.*.name, 'skip-tests') }}
+      shell: bash
+      run: |
+        echo "SKIP_TESTS=$SKIP_TESTS" >> $GITHUB_ENV
+        echo "skip-tests=$SKIP_TESTS" >> $GITHUB_OUTPUT
+
+    - name: Set Maven test options
```

Comment:
It would be nice to resolve the label earlier so that this step can be merged with the "compute-maven-options" existing step: it's already contributing to maven options